catalog/rocks/entry_catalog.go

```diff
@@ -111,15 +110,9 @@ func NewEntryCatalog(cfg *config.Config, db db.Database) (*EntryCatalog, error)
 
 	pebbleSSTableCache := pebble.NewCache(tierFSParams.PebbleSSTableCacheSizeBytes)
 	defer pebbleSSTableCache.Unref()
```

Comment:
I'm confused, it looks like the `Unref` happens before the cache is actually being closed?

Reply:
Yes.
Cache docs say to do:
```go
//   c := pebble.NewCache(...)
//   defer c.Unref()
//   d, err := pebble.Open(pebble.Options{Cache: c})
```
and here `NewPebbleSSTableRangeManager` will Ref the cache.

Reply:
And where does PebbleSSTableRangeManager Unref it?

Reply:
In the destructor?  Since we don't have  `PebbleSSTableRangeManager.Close`, we don't unref it.  I do not want to add code that is never needed to run - it will never really be tested.
I *do* need to ref the cache, though: if the same cache is passed to any other Pebble object, *that* might ref/unref it.

Reply:
I was pretty sure that `pebble.Cache` panics when you finish using it and `cache.Ref >= 1` and we use to get it in some unit test.
But I don't find evidence to it in the version that we're using - so we're good to go.